masterlist.yaml

```diff
@@ -831,6 +831,45 @@ plugins:
       - 'Order_Of_Sanctity.ESP'
       - 'Walkers of Morrowind.esp'
 
+   ###### Gameplay - Quests ######
+     ### Rise of House Telvanni ###
+  - name: 'Rise of House Telvanni.esm'
+    url:
+      - 'https://www.nexusmods.com/morrowind/mods/27545/'
+      - 'https://www.nexusmods.com/morrowind/mods/49231/'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[BOM_ROHT_Patch.ESP](https://www.nexusmods.com/morrowind/mods/49231/)' ]
+        condition: 'file("Rise of House Telvanni.esm") and file("Beautiful cities of Morrowind.ESP")'
+      - <<: *useOnlyOneX
+        subs: [ 'Rise of House Telvanni' ]
+        condition: 'file("BOM_ROHT_Patch.ESP") and file("ROHT_2_0_8.esp")'
+
+  - name: 'BOM_ROHT_Patch.ESP'
+    url:
+      - 'https://www.nexusmods.com/morrowind/mods/27545/'
+      - 'https://www.nexusmods.com/morrowind/mods/49231/'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Rise of House Telvanni' ]
+        condition: 'file("BOM_ROHT_Patch.ESP") and file("ROHT_2_0_8.esp")'
+
```

Comment:
This file isn't on the Rise of House Telvannimod page.

This can be replaced with an `inc:` in the metadata for `ROHT_2_0_8.ESP`.
```yaml
    msg:
      - <<: *useOnlyOneX
        subs: [ 'Rise of House Telvanni' ]
        condition: 'file("BOM_ROHT_Patch.ESP") and file("ROHT_2_0_8.esp")'
```

I would remove this altogether as it's not needed at the moment.
```suggestion

```